File: freepascal/99.pas
program NinetyNine;
var
  i: Integer;
begin
  for i := 100 to 1 do
  begin
    WriteLn(i, ' bottles of beer on the wall, ', i, ' bottles of beer');
    WriteLn('Take one down and pass it around, ', i-1, ' bottles of beer on the wall');
  end;
  readln;
end.

File: forth/99.fs
VARIABLE NUM
99 NUM !
: TEST
0 99
-DO
	-1 NUM +!
	i . ." bottles of beer on the wall, " i . ." bottles of beer" CR
	." Take one down and pass it around, " NUM @ . ." bottles of beer on the wall" CR
	1
-LOOP ;
TEST